packages/indexer-agent/src/indexer.ts

```diff
@@ -332,6 +335,105 @@ export class Indexer {
     }
   }
 
+  async costModels(): Promise<CostModelAttributes[]> {
+    try {
+      const result = await this.indexerManagement
+        .query(
+          gql`
+            query costModels {
+              costModels {
+                deployment
+                model
+                variables
+              }
+            }
+          `,
+        )
+        .toPromise()
+
+      if (result.error) {
+        throw result.error
+      }
+      this.logger.debug('Fetched cost models', {
```

Comment:
I'd either remove this or make it a trace log since this action isn't very important for monitoring what the agent is doing. The case that might need attention already has a log; if there isn't a cost model set for a deployment the indexer is allocated to.